### PR TITLE
[AUD-1124] Move average artwork to common

### PR DESCRIPTION
**Pull request by raymondjacobson**
### Description
Move dominant / average rgb color logic to "common"

### Dragons

Is there anything the reviewer should be on the lookout for? Are there any dangerous changes?

### How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide repro instructions & any configuration.

In react native

### How will this change be monitored?

For features that are critical or could fail silently please describe the monitoring/alerting being added.

### CODE REVIEW

File: src/common/store/average-color/slice.ts [hunk 1]
```diff
@@ -4,7 +4,8 @@ import Color from 'common/models/Color'
 import { CID } from 'common/models/Identifiers'
 import { Track } from 'common/models/Track'
 import { Nullable } from 'common/utils/typeUtils'
-import { AppState } from 'store/types'
+
+import { CommonState } from '..'
```
Comment (sliptype) [inline src/common/store/average-color/slice.ts:8]:
```suggestion
import { CommonState } from 'common/store'
```

Comment (raymondjacobson) [reply src/common/store/average-color/slice.ts:8]: thanks!!

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/rj-aud-1124

**Review by milindNshah — APPROVED** [review]
looks good!

**Pull request merged**